packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -45,9 +60,12 @@ export async function processOutboundPatientDiscoveryResps({
       results
     );
 
-    await processPatientDiscoveryProgress({ patient: patientIds, status: "completed" });
+    await processPatientDiscoveryProgress({
+      patient: patientIds,
+      status: "completed",
+      patientDemographicsDiff,
```

Comment:
There wouldnt be a need for `patientDemographicsDiff` as we can just use `patientDemographicsDiff` to determine whether or not to set this as completed or continue processing and retrigger

Reply:
No longer relevant.